frontend/src/pages/scenes/sceneForm/SceneForm.tsx

```diff
@@ -353,8 +347,17 @@ const SceneForm: FC<SceneProps> = ({ scene, callback, saving }) => {
             <TagSelect tags={scene.tags} onChange={onTagChange} />
           </Form.Group>
 
+          <NavButtons onNext={() => setActiveTab("links")} />
+        </Tab>
+
+        <Tab eventKey="links" title="Links">
+          <Row>
+            <URLInput control={control} type={ValidSiteTypeEnum.SCENE} />
+          </Row>
```

Comment:
Same as performer form.
```suggestion
        <Tab eventKey="links" title="Links" className="col-xl-9">
          <URLInput control={control} type={ValidSiteTypeEnum.SCENE} />
```